### ISSUE TITLE
Polls in wagtail

### ISSUE THREAD

**Issue by climu** [climu/openstudyroom#414]
At the moment we cannot render polls in a blog post.

We are using machina polls in forum post. It should be possible to create a poll block for [wagtail streamfield](https://docs.wagtail.io/en/v2.8.1/topics/streamfield.html#freeform-page-content-using-streamfield) reusing [machina polls](https://github.com/ellmetha/django-machina/tree/releases/0.7/machina/apps/forum_conversation/forum_polls) templates and methods.

**Comment by climu** [comment]
So we can show polls in blog posts ([here for instance](https://openstudyroom.org/blog/2020/06/05/another-lecture-jeff-choose-topic-next-thursday/)).

Sadly machina polls views will redirect to the blog post. So after voting, asking to change vote, user will come to forum post and not the blog post.

One would have to subclass [TopicPollVoteView](https://github.com/ellmetha/django-machina/blob/e6a4152a17a01d66acd51efa9deef54e1fc8b1c3/machina/apps/forum_conversation/forum_polls/views.py#L30) to override `get_success_url`.

Changing vote should be handled too as machina just add a get param `?change_vote=true`.

Anyway it's a step forward.

**Comment by climu** [comment]
So this has created some issue.

For some reasons the above change will refer to `TopicPoll` as `machina.core.db.models.TopicPoll` in migration file.

Running `migrate` will hence raise `AttributeError: module 'machina.core.db.models' has no attribute 'TopicPoll'`.

I tried changing the import in `home.models` to `from machina.apps.forum_conversation.forum_polls.models import TopicPoll` but still migration files will refer to `machina.core.db.models.TopicPoll`.

Then I tried importing abstract models as follow:
```from machina.apps.forum_conversation.forum_polls.abstract_models import AbstractTopicPoll as TopicPoll```

This does makes migrations files consistent  since they refer to `machina.apps.forum_conversation.forum_polls.abstract_models.AbstractTopicPoll` but we then have error at runtime `type object 'AbstractTopicPoll' has no attribute 'objects`.
